tests/test_anonymization_overview.py

```
@@ -0,0 +1,314 @@
+"""
+Unit tests for anonymization overview API endpoints.
+Tests the /api/anonymization/items/overview/ endpoint that returns
+FileItem interface data for videos and PDFs.
+"""
+import pytest
+from django.test import TestCase
+from django.utils import timezone
+from rest_framework.test import APIClient
+from rest_framework import status
+from endoreg_db.models import VideoFile, RawPdfFile, SensitiveMeta, Center
+from endoreg_db.models.media.video.video_file_state import VideoFileState
```

Pylint (3.3.7)

[error] No name 'VideoFileState' in module 'endoreg_db.models.media.video.video_file_state'

(E0611)